src/NHibernate/Tuple/Entity/PocoEntityTuplizer.cs

```diff
@@ -214,13 +216,18 @@ protected virtual IProxyFactory BuildProxyFactoryInternal(PersistentClass @class
 		}
 
 		public override void AfterInitialize(object entity, bool lazyPropertiesAreUnfetched, ISessionImplementor session)
```

Comment:
It should be obsoleted. Or removed, by already calling the new method in the base class.